l10n_nl_bsn/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Burgerservicenummer (BSN) for Partners',
+    'version': '13.0.1.0.1',
```

Comment:
Shouldn't the version number start at `13.0.1.0.0` again?

Reply:
Right! :+1: